db-backup/create-db-dump.sh

```diff
@@ -1,12 +1,9 @@
 #!/usr/bin/env bash
 set -euxo pipefail
 
-set +x  # Don't log credentials
-DB_URI=$(echo $VCAP_SERVICES | jq -r '.postgres[0].credentials.uri')
-set -x  # Restore logging
 echo -n "${PUBKEY}" > /app/public.key
 gpg2 --import /app/public.key
-pg_dump "${DB_URI}" --no-acl --no-owner --clean --if-exists | gzip | \
+pg_dump "${DATABASE_URL}" --no-acl --no-owner --clean --if-exists | gzip | \
```

Comment:
😍 this is a great catch!

Reply:
Nice find! Did you try a success run on Preview with this new environment variable?

Reply:
@katstevens whilst I forced the jobs to fail by constraining the memory, they did get as far as connecting to the database and starting following steps before running out of memory, so I am confident this works.

Reply:
Excellent stuff 👍